draft-yasskin-http-origin-signed-responses.md

```diff
@@ -155,9 +161,9 @@ this validity information for some period of time.
 
 The `Signature` header field conveys a list of signatures for an exchange, each
 one accompanied by information about how to determine the authority of and
-refresh that signature. Each signature directly signs the exchange's headers and
-identifies one of those headers that enforces the integrity of the exchange's
-payload.
+refresh that signature. Each signature directly signs the exchange's URL and
+headers and identifies one of those headers that enforces the integrity of the
```

Comment:
Should we clarify that they are response headers to align with your other edits?

Reply:
Yes. I'll do that in the morning before I merge.